src/CodeCheckers/LineLengthChecker.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace CodeCheckers;
+
+use Nette\Object;
+use Nette\Utils\Strings;
+
+
+
+class LineLengthChecker extends Object
+{
+
+	public static function createLineLengthChecker($maxLineLength = 120)
+	{
+		return function ($checker, $s) use ($maxLineLength) {
+			if (!$checker->is('php') && !$checker->is('phpt')) {
+				return;
+			}
+
+			$i = 1;
+			foreach (explode("\n", $s) as $line) {
+				if (Strings::length($line) > $maxLineLength && !Strings::contains($line, '__construct')) {
```

Comment:
pak to dopadne, že se to v __construct odentruje => sníží se délka řákdů ale počet závislostí bude stejný :(

upravit jsem to a je tam i __construct